kobo/worker/logger.py

```diff
@@ -71,6 +71,10 @@ def stop(self):
 
 class LoggingIO(object):
     """StringIO wrapper that also writes all data to a logging thread."""
+    __slots__ = (
```

Comment:
I have reproduced the bug on a fresh RHEL-6.9 installation.  It was sufficient to remove the above hunk (from our locally patched kobo) and restart the ```covscand``` service (the Covscan worker).

If you want to reproduce it in your environment, please use the latest ```INSTALL.el6``` howto, which I updated today such that it works out of the box.  Alternatively, I can give you access to a machine where the bug occurs and you can debug it there.